test/controllers/tasks_controller_test.rb

```diff
@@ -90,13 +83,27 @@
   # Unskip and complete these tests for Wave 3
   describe "edit" do
     it "can get the edit page for an existing task" do
-      skip
-      # Your code here
+      #Arrange
+      task_data = {
+        name: "Edit test",
+        description: "build a test for the edit method",
+        completion_date: nil,
+      }
+
+      #Act
+      get edit_task_path(task.id), params: task_data
```

Comment:
You shouldn't need to send `params` along with the edit action - all you're doing here is loading the form, not submitting data yet.